client/src/templates/Challenges/codeally/show.tsx

```diff
@@ -60,18 +60,20 @@ class ShowCodeAlly extends Component<ShowCodeAllyProps> {
     return (
       <LearnLayout>
         <Helmet title={`${blockName}: ${title} | freeCodeCamp.org`} />
-        <iframe
-          sandbox='allow-modals allow-forms allow-popups allow-scripts allow-same-origin'
-          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-          src={`http://codeally.io/embed/?repoUrl=${url}`}
-          style={{
-            width: '100%',
-            height: 'calc(100vh - 38px)',
-            overflow: 'hidden',
-            border: 0
-          }}
-          title='Editor'
-        />
+        <ReflexContainer orientation='vertical'>
```

Comment:
Using a `ReflexContainer` is a bit of a sketchy fix.  If we want to resize it, that's fine, but then we should wrap the `iframe` in a `ReflexElement` since the children of `ReflexContainer` get passed a few props to make them work together.

If we're not interested in resizing, there's no need to use react-reflex.

Reply:
Well, I don't think we need to resize anything. It fixes the sizing issue's with the Iframe though.

Reply:
I'm afraid not. It's the kind of thing that would likely get removed by accident, since it's not obvious that the container does anything.